public/gallery/index.html

```diff
@@ -15,10 +15,17 @@
 			.grid {
 				margin: 0 auto 60px;
 				visibility: hidden;
-			}
+      }
+      .header_background {
+        z-index: -1;
+        background: url("photos/home_asset_6.png") 50%;
+        opacity: 0.3;
+        height: 100px;
+        position: absolute;
+      }
 			header {
-				background: #f98728;
-				color: white;
+        /*background-color: #fff;*/
```

Comment:
Love the design change, only issue with this is the past BrickHack logo's throughout the page aren't very visible. When you get to this check-in with me and let's find a way to make this work.